package.json

```diff
@@ -3,6 +3,24 @@
   "version": "1.0.0",
   "description": "Unlock is a protocol for decentralized access control",
   "main": "index.js",
+  "private": true,
+  "workspaces": [
+    "docker/development",
+    "locksmith",
+    "paywall",
+    "rover",
+    "smart-contracts",
+    "tests",
+    "tickets",
+    "unlock-app",
+    "unlock-js",
+    "unlock-protocol.com",
+    "wedlocks"
+  ],
+  "resolutions": {
```

Comment:
See: https://yarnpkg.com/lang/en/docs/selective-version-resolutions/

Basically, this makes everything use the same version of React no matter which it actually specifies.